documentation/docs/steps/apiProviderUpload.md

```diff
@@ -0,0 +1,45 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+With this step you can adds a new entity to the API Providers.An API provider is a concept in SAP Integration Suite, API Management which defines the connection details for services running on specific hosts whose details you want to access.
```

Comment:
There are spelling errors. Did a documentation expert have a look at this?

Reply:
@AnnikaGonnermann Could you have a look as we are almost done with the review? Thanks.